python/ray/dataframe/io.py

```diff
@@ -0,0 +1,127 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from itertools import chain
+from io import BytesIO
+import os
+
+from pyarrow.parquet import ParquetFile
+import pandas as pd
+
+from .dataframe import from_pandas, ray, DataFrame
+
+
+def read_parquet(path, columns=None, npartitions=None, chunksize=None):
```

Comment:
Make sense. Where should `npartitions` and `chunksiz` goes?

Reply:
After some thought
- the best way to do it is to set default. Since that will keep the API the same and the users don't need to set worry about what arguments are needed to ray. The chunksize and npartitions are there for them to specify. 
- Example:
```python
# inside ray/dataframe/dataframe.py

DEFAULT_NPARTITIONS = 2
DEFAULT_CHUNKSIZE = 20

def set_npartition_default(n):
    global DEFAULT_NPARTITIONS
    DEFAULT_NPARTITIONS = n

def set_chunksize_default(s):
    global s
    DEFAULT_CHUNKSIZE = s

def from_pandas(df, npartitions=None, chunksize=None):
    if not npartitions and not chunksize:
        npartitions = DEFAULT_PARTITIONS
    ....
```

Reply:
I think this makes a lot of sense. I think that we have typically been creating 1x the number of virtual CPUs, but it has been manual.  Ray knows how many CPUs it can access, so we just need to have that. We can tune this a bit ourselves for the average case and leave it to users to tune the rest.